source/routing/raptor.cpp

```diff
@@ -498,6 +514,29 @@ void RAPTOR::set_valid_jp_and_jpp(uint32_t date, const std::vector<std::string>
         }
     }
 
+    // filter accessibility
```

Comment:
you may test if there are some filtered parameters in accessibilite_params first for optimization sake.